### PR TITLE
Merge dupplicated compile options for wamrc

### PR DESCRIPTION
**Pull request by no1wudi**

### CODE REVIEW

**Comment by yamt** [comment]
* i suppose core/iwasm/include should not include internal headers.
* why do you only share members, instead of the structure itself?

**Comment by no1wudi** [comment]
> * i suppose core/iwasm/include should not include internal headers.
> * why do you only share members, instead of the structure itself?

Thanks for point that, yes we can shared the structure directly and move the header to exported include path.

File: core/iwasm/include/aot_comp_option.h [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2019 Intel Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#ifndef __AOT_COMP_OPTION_H__
+#define __AOT_COMP_OPTION_H__
+
+#ifdef _AOT_EXPORT_H
+#define uint32 uint32_t
```
[the remark this answers is not in the document]
Comment (wenyongh) [reply core/iwasm/include/aot_comp_option.h:10]: Agree, use uint32_t should be OK.

**Review by wenyongh — COMMENTED** [review]
LGTM

**Pull request merged**